Sample/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NServiceBus" version="6.1.1" targetFramework="net46" />
+  <package id="Newtonsoft.Json" version="11.0.1" targetFramework="net452" />
+  <package id="NServiceBus" version="7.0.1" targetFramework="net452" />
+  <package id="NServiceBus.Newtonsoft.Json" version="2.1.0" targetFramework="net452" />
+  <package id="System.ValueTuple" version="4.4.0" targetFramework="net46" />
```

Comment:
value tuple should not be required?

Reply:
Not sure why it was added. I'll see if I can remove.